### PR TITLE
move helm chart readme to published path

### PR DESCRIPTION
**Pull request by shivamerla**

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -1 +1,168 @@
...
+
+| Key        | Value                     | Description                                                                        |
+|------------|---------------------------|------------------------------------------------------------------------------------|
+| flavor     | openshift                 | nodeSelector tweaks to prevent provisioner to run on an infra node.                |
+| podsMountDir | /var/lib/origin/openshift.local.volumes       | This is the directory where the kubelet bind mounts the volume for pods.            |
+
+## Installing the Chart
+To install the chart with the name `hpe-flexvolume`:
+```
+helm repo add hpe https://github.com/hpe-storage/co-deployments
```
Comment (drajen) [inline docs/README.md]: This points to the wrong URL

File: docs/README.md [hunk 2]
```diff
@@ -1 +1,168 @@
...
+It's recommended to create a [values.yaml](values.yaml) file and edit it to fit the environment the chart is being deployed to. Download and edit the sample file.
+
+### Platform notes
+Certain distributions demand certain tweaks to the variables for the driver and dynamic provisioner to operate correctly. See each platform for details.
+
+#### Upstream Kubernetes
+This is the default operating mode, no tweaks are needed.
+
+#### Red Hat OpenShift and OKD
+Applicable to Red Hat OpenShift 3.10 and 3.11. 4.x is not supported.
```
Comment (drajen) [inline docs/README.md]: Red Hat will support FlexVolume in 4.1 and 4.2. So should we.

File: docs/README.md [hunk 3]
```diff
@@ -1 +1,168 @@
...
+--set backend=X.X.X.X --set username=admin --set password=xxxxxxxxx
+```
+
+## Using
+To enable dynamic provisioning of volumes through the use of `PersistentVolumeClaim` API objects, a `StorageClass` needs to be declared on the cluster. Please see the [HPE CSI Driver for Kubernetes](https://github.com/hpe-storage/csi-driver) repository for the official documentation for this Helm chart. Also, it's helpful to be familar with [persistent storage concepts](https://kubernetes.io/docs/concepts/storage/volumes/) in Kubernetes prior to deploying stateful workloads.
+
+## Support
+The HPE CSI Driver for Kubernetes Helm chart is considered beta software and only works with the HPE Nimble Storage CSP. Do not use for production and do not contact HPE for support. Please file any issues, questions or feature requests [here](https://github.com/hpe-storage/co-deployments/issues). You may also join our Slack community to chat with HPE folks close to this project. We hang out in `#NimbleStorage` and `#Kubernetes` at [slack.hpedev.io](https://slack.hpedev.io/).
+
+## Contributing
```
Comment (drajen) [inline docs/README.md]: Could you remove this and consolidate at the bottom?

File: docs/README.md [hunk 4]
```diff
@@ -1 +1,168 @@
...
+We value all feedback and contributions. If you find any issues or want to contribute, please feel free to open an issue or file a PR. More details in [CONTRIBUTING.md](CONTRIBUTING.md)
+
+## License
```
Comment (drajen) [inline docs/README.md]: Could you remove this and consolidate at the bottom?

File: docs/README.md [hunk 5]
```diff
@@ -1 +1,168 @@
...
+This is open source software licensed using the Apache License 2.0. Please see [LICENSE](LICENSE) for details.
+
```
Comment (drajen) [inline docs/README.md]: Is it possible to insert a horizontal line or distinguish the two better?

File: docs/README.md [hunk 6]
```diff
@@ -1 +1,168 @@
-HPE Helm charts
+# HPE Helm charts for
+ - CSI Driver
```
Comment (drajen) [inline docs/README.md]: Is it possible to make links to the h1 tags of each chart?

File: docs/README.md [hunk 7]
```diff
@@ -1 +1,168 @@
...
+```
+
+## Uninstalling the Chart
+To uninstall/delete the `hpe-flexvolume` deployment:
+```
+helm delete hpe-flexvolume --purge
+```
+
+## Alternative install method
+
```
Comment (drajen) [inline docs/README.md]: Excessive line break

File: docs/README.md [hunk 8]
```diff
@@ -1 +1,168 @@
...
+The HPE FlexVolume Driver for Kubernetes Helm chart is supported by the respective platform team. Currently supported platforms:
+
+- HPE Nimble Storage
+
+Please file issues through the regular support channels for the particular platform. Feature requests or general questions to developers may be filed through the [GitHub issue tracker](https://github.com/hpe-storage/co-deployments) for this project.
+
+You may also join our Slack community to chat with HPE folks close to this project for inquiries not requring our immediate response. We hang out in `#NimbleStorage` and `#Kubernetes` at [slack.hpedev.io](https://slack.hpedev.io/).
+
+## Contributing
+
```
Comment (drajen) [inline docs/README.md]: Excessive line break

File: docs/README.md (hunk 1 above)
Comment (shivamerla) [reply docs/README.md]: thanks, fixed it..

File: docs/README.md (hunk 2 above)
Comment (shivamerla) [reply docs/README.md]: Its not tested with 4.1 yet Michael. Once QA is able to get it verified, we can add that..

**Comment by shivamerla** [comment]
@drajen addressed review comments, please let me know if any changes required..

File: docs/README.md (hunk 2 above)
Comment (shivamerla) [reply docs/README.md]: added those..

**Pull request merged**